Modules/Setup.bootstrap.in

```diff
@@ -25,6 +25,7 @@ _weakref _weakref.c
 
 # commonly used core modules
 _abc _abc.c
+_copy _copy.c
```

Comment:
The build rule should be either added to ``Setup.bootstrap.in`` or ``Setup.stdlib.in`` rules. Is the module required to run ``setup.py``? In case it it not required, it should not be added to the bootstrap file.

I don't think we have any documentation concerning new modules in the dev guide yet.